internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -64,6 +67,20 @@ type ClusterTopologyBuiltins struct {
 	Class string `json:"class,omitempty"`
 }
 
+// ClusterNetworkBuiltins represents builtin cluster network variables.
+type ClusterNetworkBuiltins struct {
+	// APIServerPort is the port the API Server should bind to.
+	APIServerPort *int32 `json:"apiServerPort,omitempty"`
+	// 	ServiceDomain is the network ranges from which service VIPs are allocated.
+	ServiceDomain string `json:"serviceDomain,omitempty"`
+	// ServicesCIDRBlocks is network ranges from which Service networks are allocated.
+	ServicesCIDRBlocks []string `json:"services,omitempty"`
```

Comment:
```suggestion
	// ServiceCIDRs are the network ranges from which Service VIPs are allocated.
	ServiceCIDRs []string `json:"serviceCIDRs,omitempty"`
```

Reply:
/cc @fabriziopandini @vincepri to get consensus on the variable name (same in https://github.com/kubernetes-sigs/cluster-api/pull/6171#discussion_r811348749)